selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -87,10 +88,11 @@ void Networking::attemptInitialization() {
   )");
   main_layout->setCurrentWidget(wifiScreen);
   ui_setup_complete = true;
+  wifi->requestScan();
 }
 
 void Networking::refresh() {
-  if (!this->isVisible()) {
+  if (!this->isVisible() && !wifi->firstRefresh) {
```

Comment:
just use ui_setup_complete

Reply:
So we want to request the scan immediately on start up and update the networks in the background once, so that means two calls to Networking::refresh if we use `ui_setup_complete`, and the second call won't allow the update in the background since `ui_setup_complete` will be true the second time. I'll think of a cleaner way though now that we're waiting for the scan all the time